Adds personal information to about us

CRen88888 opened the pull request:
Populates the about us page with details about me.

CRen88888 committed:
Adds personal information to about us

Populates the about us page with details about me.

jbaca1-ship-it commented:
I have three constructive critiques.
First, "NBA 2k" is a series and the name of said series is spelled as "NBA 2K" (the "k" is capitalized).
Second, a blue whale's heart weighs over 400lbs. Descriptive language is a must if you want to truly reach your audience. 
Third, you should include your SSN just for funsies ( ͡~ ͜ʖ ͡°)

CRen88888 committed:
Updates about us.txt file.

Updates aboutus.txt file to include a more recent description with responses to criticism.

Removes the file, then re-adds it, which appears to be the only way to update the file.
